tf_files/aws/modules/commons-vpc-es/variables.tf

```diff
@@ -29,3 +29,13 @@ variable "organization_name" {
   description = "For tagging purposes"
   default     = "Basic Service"
 }
+
+variable "es_version" {
+  description = "What version to use when deploying ES"
+  default     = "7.4"
```

Comment:
you want to make it `6.8` here too, just to be consistent?

Reply:
makes sense